### ISSUE TITLE
Request for evaluation script for public datasets (e.g., EchoNet) using released DISCOVR checkpoints

### ISSUE THREAD

**Issue by Abdul-Rehman-Astro** [mdivyanshu97/DISCOVR#2]
Hello authors,

Thank you for releasing the DISCOVR codebase and pretrained checkpoints. We really appreciate the effort and the contribution.

I am currently trying to verify and reproduce the reported results using the provided DISCOVR checkpoints on a public dataset (EchoNet-Dynamic), and I am facing some confusion regarding evaluation / inference.

Main issue
While the repository provides:

pretraining scripts
fine-tuning scripts
pretrained checkpoints (e.g., checkpoint-799.pth)

it is currently unclear how to perform evaluation or inference on a test dataset using these checkpoints.

Specifically, I could not find:
a dedicated evaluation script
or a clear example of inference on a dataset (e.g., EchoNet or Kinetics-style datasets)

I tried using your already given script:

run_discovr_encoder.py
The repository mentions inference via:

cd /path/to/DISCOVR

python run_discovr_encoder.py \
  --checkpoint /path/to/checkpoint-799.pth \
  --num_frames 64 \
  --batch_size 2 \
  --device cuda

How can this script be used for dataset-level inference (e.g., on EchoNet test split)?
Does it output video-level embeddings only, or classification predictions as well?
Is there a recommended way to compute evaluation metrics (AUC / accuracy / etc.) from its outputs?
